packages/bot/src/stores/run-panel-store.js

```diff
@@ -27,6 +27,8 @@ export default class RunPanelStore {
     @observable is_running            = false;
     @observable is_drawer_open        = false;
     
+    is_contract_started = false;
```

Comment:
Instead of setting this extra property, you can check if the `ContractCardStore` has a `contract` value (i.e. not `null`). This will make it clearer to understand what's happening here.